src/main/kotlin/org/rust/lang/core/completion/RsVisibilityCompletionContributor.kt

```diff
@@ -0,0 +1,15 @@
+package org.rust.lang.core.completion;
+
+import com.intellij.codeInsight.completion.CompletionContributor;
+import com.intellij.codeInsight.completion.CompletionType
+import org.rust.lang.core.RsPsiPattern
+import org.rust.lang.core.RsPsiPattern.declarationPattern
+import org.rust.lang.core.RsPsiPattern.inherentImplDeclarationPattern
+import org.rust.lang.core.or
+
+class RsVisibilityCompletionContributor: CompletionContributor() {
```

Comment:
Let's not introduce new `CompletionContributor` and use `RsCompletionContributor`. (I'd rather remove `RsKeywordCompletionContributor`)